bin/runSVUnit

```diff
@@ -48,6 +48,7 @@ my $wavedrom;
 my @directory;
 my @simargs;
 my @compileargs;
+my @elabargs;
```

Comment:
I took the liberty of renaming this, to be a bit shorter. It fits with `simargs`.